beacon-chain/rpc/eth/beacon/blocks.go

```diff
@@ -243,11 +243,11 @@ func (bs *Server) GetBlockSSZ(ctx context.Context, req *ethpbv1.BlockRequest) (*
 	return &ethpbv1.BlockSSZResponse{Data: sszBlock}, nil
 }
 
-func (bs *Server) GetBlockAltair(ctx context.Context, request *ethpbv2.BlockRequest) (*ethpbv2.BlockResponse, error) {
+func (bs *Server) GetBlockV2(ctx context.Context, request *ethpbv2.BlockRequestV2) (*ethpbv2.BlockResponseV2, error) {
```

Comment:
I renamed all `Get...Altair` to `Get...V2` (for now of course) because these endpoints return both Phase 0 and Altair data.